### ISSUE TITLE
Push notifications are not received in "not started" mode on Android

### ISSUE THREAD

**Issue by todorone** [invertase/react-native-firebase#972]
### Issue

I've set up Messaging/Notifications modules on both iOS/Android. On iOS push notifications being received in all modes(foreground/background/not started). On Android push notifications are received in foreground/background modes, but are not received when the application is not started. I've rechecked installation instructions couple of times, seems everything is ok from app side. What could be the problem?

### Environment

1. Application Target Platform: Android 5

2. Development Operating System: MacOS

3. Build Tools: XCode 9

4. React Native version: 0.54.4

5. RNFirebase Version: 4.0.1

**Comment by chrisbianca** [comment]
@todorone What type of push notifications are you sending? Data only messages, or do they have the notification part as well?  If you can include details of this I might be able to help

**Comment by todorone** [comment]
@chrisbianca Thanks for promptness.
I'm sending messages with notification part. I do it via Firebase console to be sure it's not the problem with sending. And the fact notifications come on iOS with the app unloaded. I thought maybe i set up something incorrect on Android, but couldn't find any change from instruction.

**Comment by chrisbianca** [comment]
If it has the notification part then when the app is closed it's handled by the Android OS directly rather than the RNFirebase library. Have you set the priority to `high`? What details are you sending in the notification?

**Comment by todorone** [comment]
Yeah, that's why it's so strange to me. Can't understand why Android OS is not receiving these pushes. Messages are fairly simple with high priority. And when messages are not sent, they have status `Failed`, but these messages, that do not come to Android, have status `Completed`.
<img width="727" alt="screen shot 2018-04-12 at 10 55 30 am" src="https://user-images.githubusercontent.com/423701/38663707-3f3d77ee-3e40-11e8-8979-38dcf90fc55f.png">

**Comment by chrisbianca** [comment]
Yes, very odd. From the looks of above, you're doing everything I'd expect to work.
Is it Android 5 you're on? If it's 8+ then I'd suggest trying to set up a notification channel, but don't think this is necessarily appropriate here.
